types/ramda/index.d.ts

```diff
@@ -2163,10 +2169,10 @@ export function toUpper(str: string): string;
  * list, successively calling the transformed iterator function and passing it an accumulator value and the
  * current value from the array, and then passing the result to the next call.
  */
-export function transduce<T, U>(xf: (arg: readonly T[]) => T[], fn: (acc: readonly U[], val: U) => U[], acc: readonly T[], list: readonly T[]): U;
-export function transduce<T, U>(xf: (arg: readonly T[]) => T[]): (fn: (acc: readonly U[], val: U) => U[], acc: readonly T[], list: readonly T[]) => U;
-export function transduce<T, U>(xf: (arg: readonly T[]) => T[], fn: (acc: readonly U[], val: U) => U[]): (acc: readonly T[], list: readonly T[]) => U;
-export function transduce<T, U>(xf: (arg: readonly T[]) => T[], fn: (acc: readonly U[], val: U) => U[], acc: readonly T[]): (list: readonly T[]) => U;
+export function transduce<T, U>(xf: (arg: readonly T[]) => T[], fn: (acc: readonly U[], val: U) => U[], acc: readonly T[], list: readonly T[]): U[];
```

Comment:
According R.transduce's signature of Ramda official doc: `(c → c) → ((a, b) → a) → a → [b] → a`, I think its typescript signature is not correct.

Reply:
oh... `acc` and `list` should be `readonly U[]` right?
(if you mean it should be more generic than just arrays, i can add the official signature as a fallback but i suspect that'd make it *not* error on arrays when it should)

Reply:
well... actually `U[]` can (should?) probably be replaced with `V`. not sure what else should be widened as well?

Reply:
`R.transduce` can be regard as `R.compose(R.reduce, f, g, h)`, so I think `R.transduce`'s signature may be the same as `R.reduce`'s signature.